contents/docs/product-analytics/web-analytics.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Web analytics
+sidebar: Docs
+showTitle: true
+availability:
+    free: partial
+    selfServe: partial
+    enterprise: partial
+---
+
+> Web analytics are currently in [public beta](/docs/getting-started/enable-betas). This means it's not yet a perfect experience, but we'd love to know your thoughts. Please [share your feedback](http://app.posthog.com/home#supportModal) and [follow our roadmap](https://github.com/PostHog/posthog/issues/18547).
```

Comment:
I've been calling it opt-in beta elsewhere (to distinguish between now - when people have to opt-in to see it - and the future when it's enabled for most customers but still days beta)